DataSetInfoAbstractClass.py

```diff
@@ -31,17 +31,20 @@ def load(self):
         return
 
     @abstractmethod
-    def visualize(self, randIndexes, rightDomain, right_decoded_imgs,
+    def visualize(self, modelHandle,
+                  leftPredicted, rightPredicted,
+                  rightDomain, right_decoded_imgs,
                   rightToLeftCycle, right_generatedImgs, leftToRightImgs,
                   leftDomain, left_decoded_imgs, leftToRightCycle,
-                  left_generatedImgs, rightToLeftImgs, params, n=10):
+                  left_generatedImgs, rightToLeftImgs, params, n):
         """
         Visualizes all of the data passed to it. This does not need to return
         anything.
 
         Args:
-            randIndexes (array of ints): Random points to portray,
-                                         but same for each set of data
+            modelHandle (model): Holds all the components of the model
```

Comment:
Is `model` an actual type in this system? Would be nicer to have actual expected object type if not.

Reply:
model is the actual type, I could rename the type if that would make it easier to read.

Reply:
yeah, I agree that `model` is a confusing object name